source/type/pb_converter.cpp

```diff
@@ -409,6 +413,30 @@ void fill_pb_object(nt::Line const* l, const nt::Data& data,
     }
 }
 
+void fill_pb_object(const nt::LineGroup* lg, const nt::Data& data,
+        pbnavitia::LineGroup* line_group, int max_depth,
+        const pt::ptime& now, const pt::time_period& action_period, const bool show_codes){
+    if(lg == nullptr)
+        return ;
+    int depth = (max_depth <= 3) ? max_depth : 3;
+
+    for (const auto& comment: data.pt_data->comments.get(lg)) {
```

Comment:
comments for depth = 0 ? not sure it's really useful but you know better than me :smile: